src/Compilers/VisualBasic/Test/Semantic/Semantics/TypeOfTests.vb

```diff
@@ -211,12 +409,287 @@ Module Program
     End Sub
 
 End Module
-    </file>
-</compilation>
-
-            Dim compilation = CompilationUtils.CreateCompilationWithMscorlibAndVBRuntimeAndReferences(source, , TestOptions.ReleaseExe)
-            CompilationUtils.AssertTheseDiagnostics(compilation,
-<expected>
+]]>.Value
+
+            Dim expectedOperationTree = <![CDATA[
```

Comment:
>expectedOperationTree [](start = 16, length = 21)

Please adjust tests to target specific expressions instead. Also, consider limiting testing to distinct scenarios that actually interesting from IOperation point of view.

Reply:
I am slightly confused - I thought we were specifically asked to extend existing compiler semantic tests to avoid duplicating test code and verification wherever possible. Is our preference now always to create new tests for Operation tree verification?

Reply:
> Is our preference now always to create new tests for Operation tree verification

I didn't suggest to create new tests. I suggested to make the tests more targeted and readable for humans. I.e. don't dump IOperation for arbitrary big chunks of code and expect reviewers to verify them. Why do you include entire declaration statements, ```if``` statements and the like into the base line? If I were you, in this PR I would verify IOperation only for the TypeOf expressions.

Reply:
@333fred Will your tool be able to handle updating tests written in this manner? Otherwise, we might have to hand fix all these tests everytime the tree shape changes.

Reply:
If you mean putting multiple asserts in a single test function, it won't be able to handle that. We could make it do something like that, but it would probably be more complex than useful.